dockerfiles/docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: '3'
```

Comment:
I'm getting the same warning message about the version, but I'm not sure we can remove it since we still tell people to use the Docker Compose version that's installed via `pip` in step 3 from our guide: https://dev.readthedocs.io/en/latest/install.html#set-up-your-environment

Reply:
Don't know. All of our scripts no longer use that docker-compose command anyway, we use `docker compose`. We should just remove that dependency.

Reply:
We should test if our current guide continues working before making this change, then. If it doesn't, we should update it to tell people to install the latest version of Docker that contains Docker Compose.

Reply:
Don't think we should support the old version of docker. We can just remove that dependency, we already link to the official guide installation of docker, and that works.

Reply:
As long as the guide continues working, 👍🏼

Reply:
We are not even installing docker-compose anymore with pip https://github.com/readthedocs/common/commit/9932d8a070a387211bf283dd2ff20d2f42af62ed.

Reply:
Cool